vnext/PropertySheets/ReactPatches.targets

```diff
@@ -5,44 +5,49 @@
 -->
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-  <Target Name="EvaluatePatchReactInputsOutputs" BeforeTargets="PrepareForBuild">
-    <ItemGroup>
-      <DeforkingPatchFiles Include="$(ReactNativeWindowsDir)\DeforkingPatches\**" Condition="'$(PATCH_RN)'=='true'"/>
-      <AllReactSourceFiles Include="$(ReactNativePackageDir)\**\*.cpp;$(ReactNativePackageDir)\**\*.h" />
-      <ReactSourceFiles Include="@(AllReactSourceFiles)" Condition="'$(PATCH_RN)'!='true' OR !Exists('$(ReactNativeWindowsDir)\DeforkingPatches\%(AllReactSourceFiles.RecursiveDir)%(AllReactSourceFiles.Filename)%(AllReactSourceFiles.Extension)')" />
-      <JsiReactHeaderFiles Include="$(ReactNativePackageDir)\ReactCommon\turbomodule\core\*.h" />
-    </ItemGroup>
-  </Target>
-
   <!--
     Visual Studio has its own incremental build logic for simple cases that
-    falls over here. Force use of MSBuild logic here.
+    falls over for patching. Force use of MSBuild logic.
   -->
   <PropertyGroup>
     <DisableFastUpToDateCheck>True</DisableFastUpToDateCheck>
   </PropertyGroup>
 
-  <Target Name="CopyReactSources"
-          AfterTargets="EvaluatePatchInputsOutputs"
-          BeforeTargets="PrepareForBuild">
-    <Copy SourceFiles="@(ReactSourceFiles)"
-          DestinationFiles="@(ReactSourceFiles->'$(ReactNativeDir)\%(RecursiveDir)%(Filename)%(Extension)')"
-          SkipUnchangedFiles="true" />
-  </Target>
+    <!--
+      Do not patch facebook/react-native by default until we make the switch
+      off of microsoft/react-native.
+    -->
+  <PropertyGroup Condition="'$(OSS_RN)'!='true'">
+    <PATCH_RN>true</PATCH_RN>
+  </PropertyGroup>
 
-  <Target Name="CopyDeforkingPatches"
-          AfterTargets="EvaluatePatchInputsOutputs"
-          BeforeTargets="PrepareForBuild">
-    <Copy SourceFiles="@(DeforkingPatchFiles)"
-          DestinationFiles="@(DeforkingPatchFiles->'$(ReactNativeDir)\%(RecursiveDir)%(Filename)%(Extension)')"
-          SkipUnchangedFiles="true" />
+  <Target Name="EvaluateReactPatchInputs" BeforeTargets="CopyReactPatches">
+    <PropertyGroup>
+      <ReactCommonSrcPath>$(ReactNativePackageDir)\ReactCommon\</ReactCommonSrcPath>
```

Comment:
Incorporating changes from the PR to move things to ReactCommon to avoid conflicts, since we're already making changes here.